packages/dapp-high-roller/src/components/app-root/app-root.tsx

```diff
@@ -214,7 +217,7 @@ export class AppRoot {
       pathname: "/game",
       state: {
         isProposing,
-        betAmount: "0.1"
+        betAmount: ethers.utils.formatEther(this.state.appInstance.myDeposit)
```

Comment:
Do we do any error handling before we `formatEther`?

What happens if someone enters "some amount"?

Reply:
Since the input type is `number`, this happens:

![image](https://user-images.githubusercontent.com/118913/53441776-4dadae00-39e6-11e9-9352-6e50532d70bd.png)

Reply:
Also, note that this comes from the appInstance - it can't get there without blowing up somewhere before in the stack :P